src/trumbowyg.js

```diff
@@ -452,7 +452,11 @@ jQuery.trumbowyg = {
                 t.doc.execCommand('enableObjectResizing', false, false);
             } catch (e) {
             }
-            t.doc.execCommand('defaultParagraphSeparator', false, 'p');
+            try {
+                // Disable for old IE (support from IE 11)
+                t.doc.execCommand('defaultParagraphSeparator', false, 'p');
```

Comment:
This should be in the same try catch as the execCommand above, if the first fail, the second one will fail too.